AdobeStockClientApi/Api/ConnectionObjectInterface.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
Unused interface.

Reply:
This interface consists of methods which can be used for current adobe stock integration and just added to the ConnectionAdapterInterface. This will not affect any other 3rd party implementation if the only initiateConnection method is needed to be changed via preference.